capstone/fitness/src/main/java/com/google/sps/servlets/CalendarDisplayServlet.java

```diff
@@ -12,22 +12,22 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 // CalendarDisplayServlet sends the correct information to calendar.html to display the calendar and the workout. 
 @WebServlet("/cal-display")
 public class CalendarDisplayServlet extends HttpServlet { 
   Gson gson = new Gson();
+  String eventsJson;
```

Comment:
Servlet should not have non-static vars.  Any data bound to a session should be stored in and retrieved from httpsession.

Reply:
Okay. Changing to draft PR while I make this change just bc it might take me a minute.